### PR TITLE
feat(publicURL page): #64 Designed the public url page for dashboard

### PR DESCRIPTION
**Pull request by VenkatVenkatesh**
1. Designed the public URL page.
2. Addressed the review comments for the issue #28

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Can you put a screenshot in the PR description please

File: web/src/app/services/user.service.ts [hunk 1]
```diff
@@ -48,15 +48,15 @@ export class UserService {
     );
   }
 
-  getUserSocialRecord(userId: string): Observable<UserSocial> {
+  getUserSocialRecord(userId: string) {
```
Comment (eddiejaoude) [inline web/src/app/services/user.service.ts]: Why remove the return type?

File: web/src/app/components/navbar/navbar.component.ts [hunk 2]
```diff
@@ -9,6 +10,7 @@ import { Router } from '@angular/router';
 })
 export class NavbarComponent implements OnInit {
 
+  public userName: string;
```
Comment (eddiejaoude) [inline web/src/app/components/navbar/navbar.component.ts]: Space needed between class variable and constructor

**Comment by eddiejaoude** [comment]
Looks good, couple of small comments

File: web/src/app/services/user.service.ts (hunk 1 above)
Comment (VenkatVenkatesh) [reply web/src/app/services/user.service.ts]: Removed for debugging. Will add it now.

**Comment by VenkatVenkatesh** [comment]
@eddiejaoude I have addressed the review comments. Could you please review and merge.

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**